frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -397,7 +397,8 @@ function ReaderHighlight:addToMainMenu(menu_items)
         text = _("Long-press on text"),
         sub_item_table = {
             {
-                text = _("Highlight long-press interval"),
+                text = T(_("Highlight long-press interval: %1 s"),
+                    G_reader_settings:readSetting("highlight_long_hold_threshold_s", 3)),
```

Comment:
Will the menu item be updated on changing the interval?

Reply:
No. I will change that.